test/e2e/e2e.go

```diff
@@ -11,6 +11,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	metallbiov1alpha1 "github.com/metallb/metallb-operator/api/v1alpha1"
```

Comment:
do we need this? it is already imported under metallbv1alpha

Reply:
no when created the resource it defined different API I will use the same one here and in the controller